Header header

float32[] Qdot_WF
float32[] Qdot_SF
float32[] Qdot_meas
float32[] Qdot_tang
float32[] Qdot_u
float32[] Qdot_v

